doc/source/installation.rst

```diff
@@ -0,0 +1,172 @@
+Installation
+============
+
+SIFT is provided either as Conda packages or as a compressed all-in-one
+bundle (tarball/zip) of all dependencies with a single start script.
+Unless otherwise documented below, it is recommended that the software
+bundle be used for any novice users as it requires the least amount of
+prior execution of conda or command line tools. Conda-based installation
+is best for testers, developers, and those wishing to customize versions
+of dependencies used by SIFT.
+
+System Requirements
+-------------------
+
+SIFT works on Windows, Mac, and Linux, but depends on many complex libraries
+and relies heavily on GPU (video card) processing via the PyOpenGL and VisPy
+libraries which may make installation difficult on some platforms or systems.
+In general it is best to have a local physical system to run SIFT as opposed
+to a virtual machine or remote server. Virtual machines and remote servers
+commonly have limited ways of displaying OpenGL visualizations to another
+machine.
+
+While the below system resources should work they should be considered the
+lower end of machines for running SIFT. In general the better GPU and faster
+memory (RAM) the better experience you'll have with SIFT.
+
+* Windows 10+ / Mac OS X >11.0 / Linux >= Rocky Linux 8
+* 16GB RAM
+* SSD/NVMe drive with 128GB+ available
+* GPU with 4GB VRAM and OpenGL 3+ support
+* Data files to be loaded can require up to 1TB or more
+
+Ultimately operating system support is limited to those supported by
+conda-forge as this is the primary package repository where dependencies
+are pulled from.
+
+.. _bundle-install:
+
+Application and Bundle
+----------------------
+
+Windows Bundle
+^^^^^^^^^^^^^^
+
+SIFT is distributed as a compressed `.zip` archive with all dependencies
+provided internally. The bundle (.zip) can be downloaded from our FTP server
+`here <https://bin.ssec.wisc.edu/pub/sift/dist/>`_. Once downloaded and the zip
+file extracted, the file `SIFT.bat` can be double clicked to start SIFT.
+
+By default SIFT caches files in a "workspace" located
+at the user's
+``/Users/<User>/AppData/Local/CIMSS-SSEC/SIFT/Cache/workspace`` directory.
+Configuration files for the application are stored in the user's
+``/Users/<User>/AppData/Roaming/SIFT/settings`` directory.
+
+Linux Bundle
+^^^^^^^^^^^^
+
+SIFT is available as an all-in-one tarball (``.tar.gz``) for Rocky Linux 8+
+systems. It can be downloaded from our FTP server
+`here <https://bin.ssec.wisc.edu/pub/sift/dist/>`_. Once downloaded the files
+must be extracted:
+
+.. code-block:: bash
+
+   tar -xzf SIFT_1.1.2.tar.gz
+
+This will create a ``SIFT`` directory. SIFT can then be run by doing:
+
+.. code-block:: bash
+
+   SIFT/SIFT.sh
```

Comment:
```suggestion
   tar -xzf SIFT_X.Y.Z.tar.gz

This will create a ``SIFT_X.Y.Z`` directory. SIFT can then be run by doing:

.. code-block:: bash

   SIFT_X.Y.Z/SIFT.sh
```